src/mapper/pkg/resolvers/helpers.go

```diff
@@ -73,3 +111,70 @@ func (i *intentsHolder) GetIntentsPerService(namespaces []string) map[model.Otte
 	}
 	return result
 }
+
+func (i *intentsHolder) getConfigMapNamespace() (string, error) {
+	namespace := viper.GetString(config.NamespaceKey)
```

Comment:
I find it weird that you're reading command-line arguments outside `main`. It makes this part of the code less reusable and harder to test. Instead, move the part to `main()`, and pass the namespace as an argument for `NewIntentsHolder`.

Reply:
added config, better now?